### PR TITLE
trigger for upgrade resource

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
Signed-off-by: Prajyot Parab <prajot.parab@ibm.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/195#" title="Author self-approved">Prajyot-Parab</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [Prajyot-Parab]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by bpradipt — DISMISSED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/195#pullrequestreview-558694898" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/195#" title="Author self-approved">Prajyot-Parab</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [Prajyot-Parab,bpradipt]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: modules/5_install/install.tf
Comment (yussufsh) [inline modules/5_install/install.tf:268]: Count is not needed here if I want to deploy a cluster without upgrade?

**Comment by ltccci** [comment]
New changes are detected. LGTM label has been removed.

File: modules/5_install/install.tf
Comment (Prajyot-Parab) [reply modules/5_install/install.tf:269]: Fixed.

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/195#pullrequestreview-558770813" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/195#" title="Author self-approved">Prajyot-Parab</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [Prajyot-Parab,bpradipt]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by yussufsh — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/195#pullrequestreview-558770813" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/195#" title="Author self-approved">Prajyot-Parab</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/195#pullrequestreview-558828461" title="LGTM">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [Prajyot-Parab,bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
